buildSrc/src/main/java/com/epmedu/animeal/AnimalisPlugin.kt

```diff
@@ -0,0 +1,45 @@
+package com.epmedu.animeal
+
+import com.android.build.gradle.AppPlugin
+import com.android.build.gradle.LibraryPlugin
+import com.epmedu.animeal.configure.configureAndroidApplication
+import com.epmedu.animeal.configure.configureAndroidLibrary
+import com.epmedu.animeal.configure.configureBuildFeatures
+import com.epmedu.animeal.internal.applicationExtension
+import com.epmedu.animeal.internal.libraryExtension
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.create
+
+open class AnimalisPlugin : Plugin<Project> {
```

Comment:
I guess It's better to name the plugin **AnimealPlugin**.

Reply:
Done, please check)